assets/src/design-system/components/toggle/index.js

```diff
@@ -39,6 +39,7 @@ const ICON_LEFT_POSITION = 26;
 
 const Background = styled.div(
   ({ theme }) => css`
+    box-sizing: border-box;
```

Comment:
Do we have a global style for this?
```
* {
  box-sizing: border-box;
}
```

Reply:
In the theme:

https://github.com/google/web-stories-wp/blob/0f4be34821acb3c9cef44e5dcc66d5bfc0282641/assets/src/edit-story/theme.js#L33-L37

Reply:
This was to line up the design system with the app. Once we add it as a global style in the design system we should remove this

Reply:
Moved these changes to this pr: https://github.com/google/web-stories-wp/pull/6779. That pr should align the design system to use border-box sizing